parser-typechecker/src/Unison/Runtime/IOSource.hs

```diff
@@ -959,3 +966,38 @@ syntax.docFormatConsole d =
     Special sf -> Pretty.lit (Left sf)
   go d
 |]
+
+type Note = Result.Note Symbol Ann
+
+type TFile = UF.TypecheckedUnisonFile Symbol Ann
+
+type SynthResult =
+  Result.Result
+    (Seq Note)
+    (Either (UF.UnisonFile Symbol Ann) TFile)
+
+type EitherResult = Either String TFile
+
+showNotes :: (Foldable f) => String -> PrintError.Env -> f Note -> String
+showNotes source env =
+  intercalateMap "\n\n" $ PrintError.renderNoteAsANSI 60 env source Path.absoluteEmpty
+
+decodeResult ::
+  String -> SynthResult -> EitherResult --  String (UF.TypecheckedUnisonFile Symbol Ann)
```

Comment:
Is this comment meant to be here?

Reply:
No it isn't. But this is some code I forgot to clean up. It is copied from `parser-typechecker/tests/Unison/Test/UnisonSources.hs` because the `base.u` typechecking failures weren't printed initially (leaving me very confused about what was going wrong!).

So, this `decodeResult` stuff should be moved to some appropriate place where it can be imported by `IOSource` and the `UnisonSources` test module I pulled it out of.